html/changelogs/dannno-gavel.yml

```diff
@@ -0,0 +1,6 @@
+author: Dannno
+
+delete-after: True
+
+changes: 
+  - rscadd: "Added a hammer and gavel. Court is now in session."
```

Comment:
fuuuuuuck 'eeemmmmmmmm
in all seriousness hopefully a mapper will put them on the map